src/roman_datamodels/testing/factories.py

```diff
@@ -383,6 +383,9 @@ def create_ref_meta(**kwargs):
         "description": _random_string("Reference description "),
         "exposure": {
             "type" : "WFI_IMAGE",
+            "ngroups" : 10,
+            "nframes" : 4,
+            "groupgap" : 7
```

Comment:
Can there be based on an actual MA table? I'd suggest one from the example tables from https://innerspace.stsci.edu/pages/viewpage.action?pageId=263655630

Reply:
Fixed with numbers from INS.